# Raffinis_ACE2

Online supplementary for *Genetic diversity in horseshoe bat ACE2 and sarbecovirus spike proteins mutually shape one another*


### Figure 1

* **ACE2 phylogenetic tree (Left-hand)**
  The original Maximum likelihood tree of _Rhinolophus affinis_ ACE2 CDS sequences is available [here](Figure_1/RaACE2_cds_1-1686_Rsh-out.treefile). 
  The tree rerooted by the *R. shameli* outgroup is available [here](Figure_1/RaACE2_cds_1-1686_Rsh-out_rt.nwk).

* **ACE2 CDS alignment**
  The CDS alignment of _Rhinolophus affinis_ ACE2 for constructing the left-hand tree is available [here](Figure_1/RaACE2_cds_1-1686_Rsh-out.fas). 

* **Tree with branch-specific mutation annotations**
  The rooted tree annotated with branch-specific nucleotide substitutions by TreeTime is available [here](Figure_1/treetime_mut_annotations/RaACE2_cds_1-1686_Rsh-out_annotated_tree.nexus). 
  
* **Sequence metadata**
	The metadata for all *R. affinis* ACE2 sequences are available [here](Figure_1/RaACE2_metadata.tsv). 
  
  

### Figure 2
* **Recombination and selection analysis**
  The recombination analysis result by GARD using [_Rhinolophus affinis_ ACE2 CDS alignment](Figure_2/RaACE2_cds_aln.fas) is available [here](Figure_2/RaACE2_cds_GARD-recombination.json).
  The segmented alignments ([left-hand](Figure_2/RaACE2_cds_1-1686.fas) and [right-hand](Figure_2/RaACE2_cds_1687-2415.fas)) based on the detected recombination breakpoint
  were used for selection analysis by MEME and the results are available here: [left-hand](Figure_2/RaACE2_cds_1-1686_MEME-selection.json) and [right-hand](Figure_2/RaACE2_cds_1687-2415_MEME-selection.json).

* **Infectivity assay** 
  The normalized relative infectivity values of all 36 sarbecoviruses spikes infecting HOS-ACE2/TMPRSS2 cell lines are available [here](Figure_2/sarbeco_RaACE2_infectivity_raw.xlsx).
  The mean value for each _Rhinolophus affinis_ ACE2 genotype is available [here](Figure_2/RaACE2_mean_infectivity_z-score.tsv).

* **Linear regression analysis**
  Results of OLS linear regression of which ACE2 sites explain changes in infectivity (for individual viruses and summarised) are available [here](Figure_2/infectivity_results_OLS_all). Furthermore, The result only including 11 sarbecoviruses and ACE2 mutations with significant differences in infectivity change is summarized [here](Figure_2/sarbeco_ACE2mut_inferred_infectivity_change_OLS.tsv).

### Figure 3
* **Infectivity assay with the Ra22QT77 and its mutants**
  The normalized relative infectivity values of Ra22QT77 and all mutant spikes infecting HOS-ACE2/TMPRSS2 cell lines are available [here](Figure_3/Ra22QT77mut_RaACE2_infectivity_raw.xlsx).

* **Neutralization assay with Ra22QT77 and its mutants**
  The log10 NT50 values of neutralization assay using Ra22QT77 murine sera against Ra22QT77 and its mutants are available [here](Figure_3/Ra22QT77mut_RaACE2_neutralization_summary.tsv).

### Figure 4
* **Ra22QT77-RaACE2 cryo-EM structure**
  The structure of Ra22QT77 RBD bound to _Rhinolophus affinis_ ACE2 (V-Ra-10) is available [here](Figure_4/Ra22QT77-RaACE2_PDB_9X23.pdb).

* **AlphaFold3 prediction of Ra22QT77 ins**
  The struccture of the Ra22QT77 ins RBD predicted with AlphaFold3 is available [here](Figure_4/Ra22QT77ins_AF3_model_0.cif).
  The superimposed co-structure of the predicted Ra22QT77 ins RBD and the cryo-EM resolved _Rhinolophus affinis_ ACE2 (V-Ra-10) is available [here](Figure_4/Ra22QT77ins_AF3_RaACE2_superimposed_costructure.pdb).


### Figure S1
* The rooted maximum likelihood tree of _Rhinolophus affinis_ mitochondrial cytochrome B (CYTB) sequences is available [here](Figure_S1/RaCYTB_nt_Rsh-out.treefile). The [CYTB sequences alignment](Figure_S1/RaCYTB_nt_aln_Rsh-out.fas) for construction and [CYTB metadata](Figure_S1/RaCYTB_metadata.tsv) for annotation are available here.


### Figure S2
* **Phylogenetic trees (right-hand and full alignment)**
  The maximum likelihood trees of _Rhinolophus affinis_ ACE2 CDS sequences are as follows: [right-hand tree](Figure_S2/RaACE2_cds_1687-2415_Rsh-out.treefile) and [full ACE2 tree](Figure_S2/RaACE2_cds_1-2415_Rsh-out.treefile).
  Trees rooted by the *R. shameli* outgroup are available as follows: [right-hand one](Figure_S2/RaACE2_cds_1687-2415_Rsh-out_rt.nwk) and [full ACE2 tree](Figure_S2/RaACE2_cds_1-2415_Rsh-out_rt.nwk).

* **ACE2 CDS alignments**
  The CDS alignment of _Rhinolophus affinis_ ACE2 for constructing the [right-hand tree](Figure_S2/RaACE2_cds_1687-2415_Rsh-out.fas)
  and [full ACE2 tree](Figure_S2/RaACE2_cds_1-2415_Rsh-out.fas) are available here.


* **Trees with branch-specific mutation annotations**
  The rooted trees annotated with branch-specific nucleotide substitutions by TreeTime is available here: [right-hand tree](Figure_S2/treetime_mut_annotations_1687-2415) and
  [full ACE2 tree](FIgure_S2/treetime_mut_annotations_1-2415).


